src/pybind/mgr/cephadm/module.py

```diff
@@ -630,6 +643,8 @@ def __init__(self, *args: Any, **kwargs: Any):
             self.default_cephadm_command_timeout = 0
             self.cephadm_log_destination = ''
             self.oob_default_addr = ''
+            self.ssh_keepalive_interval = 0
+            self.ssh_keepalive_count_max = 0
```

Comment:
This burned me during the smb module development, so I'll double check; this module some how automagically binds an option to a class attribute?

Reply:
We discussed this topic in an orch statndup meeting. It should be fine in the cephadm mgr module.